src/artm/core/dictionary.h

```diff
@@ -46,7 +46,7 @@ class Dictionary {
   std::vector<DictionaryEntry> entries_;
   std::unordered_map<Token, int, TokenHasher> token_index_;
   std::unordered_map<int, Token> index_token_;
-  std::unordered_map<int, std::unordered_map<int, int> > cooc_values_;
+  std::unordered_map<int, std::unordered_map<int, std::vector<float> > > cooc_values_;
```

Comment:
True, I haven't thought about this. Then how about this solution:
1. Add a new format into CollectionParserConfig.Format enum (I vote for calling it "Cooccurrence", but any reasonable name will do)
2. Treat 'docword_file_path' as concurrence file (e.g. with 3 columns)
3. Output dictionary file with coocurrences parsed from docword_file_path
4. Do NOT produce any batches